Makefile.linux.mk

```diff
@@ -98,6 +98,8 @@ build/frida-%/lib/pkgconfig/capstone.pc: build/frida-env-%.rc build/capstone-sub
 			*-armhf)  capstone_archs="arm"     ;; \
 			*-armeabi)capstone_archs="arm"     ;; \
 			*-arm64)  capstone_archs="aarch64" ;; \
+			*-mips)   capstone_archs="mips" ;; \
+			*-mipsel) capstone_archs="mips" ;; \
```

Comment:
Since we're going to the trouble of aligning the `;;` on the lines above we should also do that here.